kickoff/jsonexportl10n.py

```diff
@@ -0,0 +1,94 @@
+import os
+from collections import defaultdict
+from os import path
+import json
+
+from flask import render_template
+
+from kickoff import app
+from kickoff import config
+
+from mozilla.release.l10n import parsePlainL10nChangesets
+
+from kickoff.model import getReleases
+from kickoff.model import getReleaseTable
+from jsonexport import myjsonify
+
+
+@app.route('/json/l10n/<releaseName>.json', methods=['GET'])
+def l10nExport(releaseName):
+    # Export the l10n changeset for a product
+    releaseTable = getReleaseTable(releaseName)
+    release = releaseTable.query.filter_by(name=releaseName).first()
+
+    if release is None or release.l10nChangesets == "legacy":
```

Comment:
What does "legacy" mean here? Some kind of special word? I'm not a big fan of this approach, but it'd be better to move to a CONST variable.